flexget/entry.py

```diff
@@ -10,6 +11,23 @@
 logger = logger.bind(name='entry')
 
 
+class EntryState(Enum):
+    ACCEPTED = 'accepted'
+    REJECTED = 'rejected'
+    FAILED = 'failed'
+    UNDECIDED = 'undecided'
+
+    @property
+    def color(self) -> str:
+        return {self.ACCEPTED: 'g', self.REJECTED: 'r', self.FAILED: 'R', self.UNDECIDED: 'd',}[
```

Comment:
I think we could afford full names here rather than abbreviations, just for clarity's sake. I had to look up what 'd' was.

Reply:
I agree, although it was worth it to see you write that you had to look up the D 😆